platform-sdk/swirlds-cli/pcli.sh

```diff
@@ -174,11 +166,15 @@ run () {
   java "${JVM_ARGS[@]}" -cp "${JVM_CLASSPATH}" $MAIN_CLASS_NAME "${PROGRAM_ARGS[@]}"
 }
 
+colorize () {
+  java "${JVM_ARGS[@]}" -cp "${JVM_CLASSPATH}" com.swirlds.cli.utility.StdInOutColorize "${PROGRAM_ARGS[@]}"
+}
```

Comment:
I wouldn't pass in any of the custom JVM args here. The most common JVM arg is to increase the memory, and we don't really need to give the colonizer extra memory.

Reply:
Removed